crates/wash-cli/src/keys.rs

```diff
@@ -47,12 +46,32 @@ pub enum KeysCliCommand {
 
 pub fn handle_command(command: KeysCliCommand) -> Result<CommandOutput> {
     match command {
-        KeysCliCommand::GenCommand { keytype } => generate(&keytype),
+        KeysCliCommand::GenCommand { keytype } => {
+            let kt = keytype_parser(&keytype)?;
+            generate(&kt)
+        }
         KeysCliCommand::GetCommand { keyname, directory } => get(&keyname, directory),
         KeysCliCommand::ListCommand { directory } => list(directory),
     }
 }
 
+pub fn keytype_parser(keytype: &str) -> Result<KeyPairType> {
+    match keytype.to_lowercase().as_str() {
+        "account" => Ok(KeyPairType::Account),
+        "user" => Ok(KeyPairType::User),
+        "module" => Ok(KeyPairType::Module),
+        "actor" => Ok(KeyPairType::Module),
```

Comment:
Nit: this is totally fine as is, but in case you haven't seen it before, you can use `|` in match statements, like:
```suggestion
        "module" | "actor" => Ok(KeyPairType::Module),
```

Reply:
[this is how I felt when I realized I could have done that](https://media.giphy.com/media/MqRo4XYqR5txC/giphy.gif). 
Thanks!